server/class-component-themes-api.php

```diff
@@ -35,6 +35,18 @@ public static function fetch_required_api_endpoint( $endpoint ) {
 		if ( 200 !== $response->get_status() ) {
 			return null;
 		}
-		return $response->get_data();
+		$data = $response->get_data();
+		$data = self::sanitize( $endpoint, $data );
+
+		return $data;
```

Comment:
Could this just be `return self::sanitize( $endpoint, $response->get_data() );`?